docs/core/deploying/native-aot/interop.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Native code interop
+description: Learn about native code interop with native AOT.
+author: MichalStrehovsky
+ms.author: michals
+ms.date: 05/17/2023
+---
+# Native code interop
+
+Native code interop is a technology that allows you to access structs and functions in unmanaged libraries from managed code, or expose structs and functions in managed libraries to unmanaged code (the opposite direction).
```

Comment:
```suggestion
Native code interop is a technology that allows you to access unmanaged libraries from managed code, or expose managed libraries to unmanaged code (the opposite direction).
```

Reply:
This was copied from https://learn.microsoft.com/en-us/dotnet/standard/native-interop/pinvoke but I agree it's odd wording.